tests/pipeline/rasterization/mesh/component-write.slang

```diff
@@ -0,0 +1,49 @@
+// component-write.slang
+
+// This tests that writing to individual components of the output struct works
+
+//TEST:CROSS_COMPILE:-target spirv -profile glsl_450+spirv_1_4 -entry main -stage mesh
```

Comment:
We generally prefer render/compute tests that is runnable.
Admittedly we don't have mesh support in gfx so that is not possible yet.

The problem with cross compile tests is that they are very brittle to changes in IR optimization passes or downstream compiler updates.
If we are going to utilize more cross compile tests, we really need to start integrating llvm file-check and start writing more robust regex rules to check the resulting code.

Reply:
Yeah, it wasn't possible trivially

https://github.com/shader-slang/slang/issues/2520